{"name": "test_vwologin", "status": "broken", "statusDetails": {"message": "AttributeError: 'WebDriver' object has no attribute 'find'", "trace": "setup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"f87475cc82dbaa231a4a6675a82bfb94\")>\n\n    @allure.epic(\"tc1: Positive testcase\")\n    @allure.feature(\"vwoLogin\")\n    def test_vwologin(setup):\n        driver = setup\n        loginpage = Loginpage(driver)\n>       loginpage.loign_page(user=\"contact+atb5x@thetestingacademy.com\", pwd=\"ATBx@1234\")\n\ntests\\vwoLogintests\\test_vwoLogin.py:23: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ntests\\pageObject\\loginpage.py:31: in loign_page\n    self.get_user().send_keys(user)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <tests.pageObject.loginpage.Loginpage object at 0x00000212A088FE90>\n\n    def get_user(self):\n>       return self.driver.find.element(*Loginpage.username)\nE       AttributeError: 'WebDriver' object has no attribute 'find'\n\ntests\\pageObject\\loginpage.py:19: AttributeError"}, "start": 1710501764996, "stop": 1710501764998, "uuid": "9c53d4a6-289c-4330-b92a-806ea67fa577", "historyId": "6700d9bf3dc42d27bb0c525b9c1730a0", "testCaseId": "6700d9bf3dc42d27bb0c525b9c1730a0", "fullName": "tests.vwoLogintests.test_vwoLogin#test_vwologin", "labels": [{"name": "feature", "value": "vwoLogin"}, {"name": "epic", "value": "tc1: Positive testcase"}, {"name": "parentSuite", "value": "tests.vwoLogintests"}, {"name": "suite", "value": "test_vwoLogin"}, {"name": "host", "value": "CMI109-Dipanshu"}, {"name": "thread", "value": "10800-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.vwoLogintests.test_vwoLogin"}]}